templates/install_puppet.bat.erb

```diff
@@ -0,0 +1,11 @@
+:wait_for_pid
+FOR /F "tokens=*" %%A IN ('tasklist /FI "PID eq <%= @puppet_agent_pid %>"  /NH') DO set _task=%%A
+echo %_task% | findstr "No tasks are running" >nul
+IF NOT %errorlevel% == 0 ( GOTO wait_for_pid )
+
+
+msiexec.exe /qn /i <%= @_msi_location %> /l*v <%= @_logfile %>
```

Comment:
@kylog @MikaelSmith @cyberious I think we are setting up support for failure if we don't at least give the customer some cursory information about what went wrong with the install. 

First of all the customer is not even going to know we logged because this isn't notifying them (debug messages don't count). They are going to hit up support if they are a customer and support is going to ask them for a file. If instead we did a little bit of work ahead and told them we were logging to `x`, they at least have a file they can provide to support right out of the gate, or even better, figure it out without even having to open a support ticket.